File: practical03/Loop.f90
program loop
! Declare variables 
        integer (kind=4) :: i,n
        real (kind=4) :: integral_sum,a,b,delta

! Initialise 4-byte integer
        i = 1
        n = 12

! Initialise 4-byte float
        a = 0.0
        b = 4.D0*DATAN(1.D0)/3
        delta = (b-a)/(n-1)        

        integral_sum = tan(a) + tan(b)
        do i = 1,n-2,1
                integral_sum = integral_sum + 2 * tan(i * delta)
        end do
        integral_sum = integral_sum * (b-a) / (2 * n)
        write(6,*) 'Integral of tan(x) from 0 to pi/3 using trapezoidal rule and 11 equidistant points =',integral_sum
        write(6,*) 'log(2) =',log(2.0)
end program loop

File: practical03/Loop.c
#include <stdio.h>
#include <math.h>

int main(void) {
/* Declare variables */
	float integral_sum,a,b,delta;
	int i,n;

/* Intialise 4-byte floats */	
	a = 0.0;
	b = M_PI/3.0;
	n = 12;
	delta = (b-a)/n;
	
	integral_sum = tan(a) + tan(b);
	for (i=1; i<n; i++) {
		integral_sum = integral_sum + 2 * tan(i * delta);
	}
	integral_sum = integral_sum * (b-a) / (2*n);
	printf("Integral of tan(x) from 0 to pi/3 using trapezoidal rule and 11 equidistant points = %f\n",integral_sum);
	printf("Log(2) = %f\n",logf(2.0));
}
